src/Tutorials/Basics/HDA_flowsheet_solution_testing.ipynb

```diff
@@ -950,9 +987,118 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 42,
```

Comment:
All this output appears to be new - it looks like the previous version had not been run (i.e. the cells had been cleared before uploading). Could you fix that please.